simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in

```diff
@@ -944,11 +944,11 @@ XMLBlobs:
   - |
         <link name="base_link" />
   - |
-        <joint name="base_fixed_joint" type="fixed">
+        <joint name="base_link_fixed_joint" type="fixed">
           <origin xyz="0 0 0" rpy="0 -0 0" />
           <axis xyz="0 0 0" />
-          <parent link="base_link" />
-          <child link="root_link" />
+          <parent link="root_link" />
+          <child link="base_link" />
```

Comment:
Possibly we should document this decision somewhere more visible than in the issues?

Reply:
I plan to add a CHANGELOG.md in icub-models as done for all the other repos.